### DISCUSSION TITLE
export time entries

### DISCUSSION THREAD

**Discussion by Hannes-III** [te-online/timemanager#121] [General]
Hello, thank you all for this plugin.

The Last three weeks I was testing timemanager and I like it a lot, it fulfills my needs in tracking my hours.
To integrate it into my workflow, i would need to export the time entries so that I can continue to process it in further Calculation.

I looked through all the Issues and Discussions and found no similar request somewhere, so that I think that it is possible somehow from the UI, but I have not found jet. What do others do with the information gathered by timemanager?

I know SQL and could export it on my own from the Database if necessary. Could you please point me to the tables involved?

Thank you for your support,
Hannes

**Comment by te-online** [comment]
Hey Hannes 👋 

You can export time entries from the reports page (select a date range first). It is optimized for editing in Excel, though, not for machine reading. 🤓 

You can find names of all database tables in the initial migration file: https://github.com/te-online/timemanager/blob/main/lib/Migration/Version2003Date20210714185329.php

Let me know if this helps you. I'd really like to have a full-fledged export/import featureset, but right now that's not really on the table unfortunately.

**Reply by Hannes-III** [reply]
YES! I knew, that I could find it somewhere, but did not know where to look.
Thank you for your help, I found it, and it works!

Kind regards, Hannes
